tests/test_kci.py

```diff
@@ -210,16 +220,16 @@ def test_given_gaussian_data_and_polynomial_kernel_when_perform_kernel_based_tes
     Y = X + X1 + 0.5 * np.random.randn(300, 1)
     Z = Y + 0.5 * np.random.randn(300, 1)
 
-    assert kci.ind(X, X1, kernel_X=polynomial_kernel, kernel_Y=polynomial_kernel).pvalue > 0.05
-    assert kci.ind(X, Z, kernel_X=polynomial_kernel, kernel_Y=polynomial_kernel).pvalue < 0.05
+    assert kci.ind(X, X1, kernel_X="polynomial", kernel_Y="polynomial").pvalue > 0.05
+    assert kci.ind(X, Z, kernel_X="polynomial", kernel_Y="polynomial").pvalue < 0.05
```

Comment:
Just to make sure, it is still possible to provide a callable here, right?

Reply:
Yes, see https://github.com/py-why/pywhy-stats/pull/19/files#r1287466282

Reply:
Do you mean `def my_custom_kernel(X)`? It is tested above and still works, but as we discussed in the [other comment](https://github.com/py-why/pywhy-stats/pull/19/files#r1287477665), I can refactor the inner-workings.

I'll add this soon

Reply:
Done.